_includes/faqs.html

```diff
@@ -1,93 +1,15 @@
 <section id="faqs">
     <div class="container">
         <h2 class="section-heading text-center">FAQs</h2>
-        <!-- Agency theme hacked a bunch of the other themes so if I try to update them to something newer, the current
-        stuff breaks. I don't have time to debug why now so just using the hacked bootstrap theme agency is using. -->
         <div class="row">
-            <div class="col-6 col-md-6">
+            {% for faq in page.faqs %}
```

Comment:
I didn't know you could do this, this is awesome. Is there a way to append something in to matter from md?

Reply:
I don't think so, but there are different places you can put text content, like the `_data` directory.

Reply:
I wonder if we should just have faqs as text data in _data instead of inline like this.